### ISSUE TITLE
I got an realloc error while training tiny yolov3 on my cpu computer.

### ISSUE THREAD

**Issue by duckrider** [AlexeyAB/darknet#4725]
![command prompt error screen](https://user-images.githubusercontent.com/42626578/72664668-8b387280-39ce-11ea-98c7-74af839d08bf.png)

I got this on my command line:
 30: -nan, -nan avg loss, 0.000000 rate, 136.753000 seconds, 1920 images
Resizing, random_coef = 1.40
416 x 416
Realloc error

I'm new to neural networks so I'm not sure what could be the problem. I was training on my own dataset and using new random weights. I'm using darknet_no_gpu.

This is my cfg if it helps https://drive.google.com/file/d/18_4XK3Le87noZLXahdOeJqQT8EtytGO8/view?usp=sharing

**Comment by AlexeyAB** [comment]
How many CPU-RAM do you have?
Train by using GPU.

**Comment by LookUFOwxy** [comment]
> How many CPU-RAM do you have?
> Train by using GPU.

I just have the same problem, is that mean CPU-RAM is not enough ? besides train by GPU , are there other ways to avoid this problem?

**Comment by duckrider** [comment]
I solved this error by just using pretrained weights. I got this error from training from scratch. I used the instructions here [darknet.exe detector train data/obj.data yolov3-tiny-obj.cfg yolov3-tiny.conv.15](url) and ran their command:

> darknet.exe detector train data/obj.data yolov3-tiny-obj.cfg yolov3-tiny.conv.15

**Comment by LookUFOwxy** [comment]
> I solved this error by just using pretrained weights. I got this error from training from scratch. I used the instructions here [darknet.exe detector train data/obj.data yolov3-tiny-obj.cfg yolov3-tiny.conv.15](url) and ran their command:
> 
> > darknet.exe detector train data/obj.data yolov3-tiny-obj.cfg yolov3-tiny.conv.15

I have used pretrained weights, and  changed the content of  yolov3-tiny-obj.cfg according to the requirements，sad :(

**Comment by duckrider** [comment]
Actually I did this project 3 months ago, I just remembered what I did to fix it It wasn't from not using pretrained weights. Sorry lol. I changed both of the random=1 lines in the yolov3-tiny-obj.cfg file to random=0. Try that. I believe that was what fixed it for me.

**Comment by LookUFOwxy** [comment]
> Actually I did this project 3 months ago, I just remembered what I did to fix it It wasn't from not using pretrained weights. Sorry lol. I changed both of the random=1 lines in the yolov3-tiny-obj.cfg file to random=0. Try that. I believe that was what fixed it for me.

Thanks a lot!!! It works !!! But I still dont understand why
